cmd/lncli/commands.go

```diff
@@ -1177,7 +1179,7 @@ func monowidthColumns(words []string, ncols int) []string {
 }
 
 func create(ctx *cli.Context) error {
-	ctxb := context.Background()
+	ctxc := getContext()
```

Comment:
nit: convention is naming it `ctx`

Reply:
Ah, true! :) 

Maybe use `ctxt`?

Reply:
While I'm at it, I refactored it to use conventional naming in 6b5d165 and 019dec2 and it turns out to be not that big a diff, so if it's ok, this can be merged too.

Reply:
Variable renaming is pretty easy when you have a static type system:)